terraform/modules/hub/files/tasks/hub-config-fargate.json

```diff
@@ -24,12 +42,12 @@
       }
     ],
     "logConfiguration": {
-      "logDriver": "awslogs",
+      "logDriver": "awsfirelens",
       "options": {
-        "awslogs-group": "${deployment}-hub",
-        "awslogs-region": "${region}",
-        "awslogs-stream-prefix": "config-nginx",
-        "awslogs-create-group": "true"
+        "Name": "es",
+        "Host": "https://${logit_elasticsearch_url}:443",
```

Comment:
Do we need `:443` if we use `https://`?

Reply:
TLS support is currently disabled by default. Can we enable TLS support?

Reply:
This is just the string copied directly from the journalbeat config. We probably can get rid of the port, but I think TLS support is already enabled?

Reply:
By default, tls is set to off. Source: https://docs.fluentbit.io/manual/pipeline/outputs/tcp-and-tls

Reply:
Hm okay yes, we need to get rid of https:// and :443 here and set that.